src/libraries/Microsoft.XmlSerializer.Generator/pkg/build/Microsoft.XmlSerializer.Generator.targets

```diff
@@ -29,15 +30,28 @@
     <WriteLinesToFile Condition="'@(CscRspFile)'!=''" File="$(_CscRspFilePath)" Lines="@(CscRspFile, '%0A')" Overwrite="true" ContinueOnError="true" />
   </Target>
 
+  <Target Name="GenerateAssemblyVersionInfo" BeforeTargets="GenerateSerializationAssembly">  
+    <GetAssemblyIdentity AssemblyFiles="$(IntermediateOutputPath)$(AssemblyName)$(TargetExt)" > 
+      <Output TaskParameter="Assemblies" ItemName="SerializationAssemblyVersionInfo" />
+    </GetAssemblyIdentity>
+    <WriteLinesToFile File="$(_SerializerCsAssemblyInfoIntermediateFolder)" 
+      Lines="using System.Reflection%3B
+
+      [assembly: AssemblyVersion( &quot;%(SerializationAssemblyVersionInfo.Version)&quot; )]
+      [assembly: AssemblyFileVersion( &quot;%(SerializationAssemblyVersionInfo.Version)&quot; )]"
+      Overwrite="true"
```

Comment:
Does SerializationAssemblyVersionInfo have a separate attribute for AssemblyFileVersion? What if the main assembly has a different AssemblyVersion and AssemblyFileVersion?

Reply:
I took a look and there's no way to retrieve this so probably best to leave this as it is.

Reply:
Thanks for the feedback!  I didn't see a way either.